src/components/GoalEditForm/GoalEditForm.tsx

```diff
@@ -24,7 +24,16 @@ interface GoalEditFormProps {
 const GoalEditForm: React.FC<GoalEditFormProps> = ({ goal, onSubmit }) => {
     const locale = useLocale();
     const [busy, setBusy] = useState(false);
-    const update = useGoalUpdate(goal.id);
+    const { helpers, onUpdateGoal } = useGoals(
```

Comment:
can we directly export invalidate function? Try not use general titles like `helpers`.